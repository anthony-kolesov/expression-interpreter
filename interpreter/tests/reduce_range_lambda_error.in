out reduce({1, 3}, 0, theta nu -> {7, 8})
