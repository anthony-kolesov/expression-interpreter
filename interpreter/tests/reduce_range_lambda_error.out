ERROR:1:Can't return vector value as result of lambda function.
